cubic-server/registry/Biome.hpp

```diff
@@ -110,13 +110,13 @@ inline void registry::setupDefaultsBiome(registry::Biome &registry)
         .name("cubsrv:my_super_cool_biome_lol_haha")
         .id(1)
         .precipitation("none")
-        .temperature(0.8)
-        .downfall(0.4)
+        .temperature(0)
```

Comment:
Why did you change this biome knowing that it does not appear on generation ?

Reply:
So I can see that it, in fact, does not appear